doc/070_encryption.rst

```diff
@@ -49,3 +49,51 @@ per repository. In fact, you can use the ``list``, ``add``, ``remove``, and
     ----------------------------------------------------------------------
      5c657874    username    kasimir   2015-08-12 13:35:05
     *eb78040b    username    kasimir   2015-08-12 13:29:57
+
+**************************
+Using master key from file
+**************************
+
+The flag ``--masterkeyfile`` allows you to directly use a file for the master key. Then no key needs to be stored
+in the repository. Make sure your masterkey cannot be accessed by unauthorized persons and that you  have a backup
+of you masterkey file ready (e.g. print-out)!
+
+
+.. code-block:: console
+
+    $ restic -r /srv/restic-repo --masterkeyfile master.key init
+    created restic repository f855d38126 at /tmp/repo
+
+    Please note that you need the masterkey tmp.master to access the repository
```

Comment:
[typo?] Did you mean `master.key` instead of `tmp.master` here?

Reply:
is now corrected. Thanks!